utils/utils.py

```diff
@@ -194,7 +206,11 @@ class QuitMessage:
 
 class Worker(threading.Thread):
     def __init__(self, queue: Queue):
-        super().__init__(target=self._do_work, args=(queue, ))
+        # [KN] We mark the thread as daemonic to make sure it will
```

Comment:
I am assuming that these are initials of name.
Sorry but I am not  aware of. Do we generally enter our initials of name in code in this project?

Reply:
I prefer to leave my initials in the comments that I can consider as important. But I can remove them if they look like a noise.

Reply:
I don't see a problem. But generally I have seen hard restrictions in many projects related to this, so was just curious.

Reply:
Ok to keep initials I think